gallery/src/components/NxDropdown/NxDropdownPage.tsx

```diff
@@ -133,21 +129,11 @@ const NxDropdownPage = () =>
       <CodeExample content={nxDropdownNavigationExampleCode} />
     </GalleryTile>
 
-    <GalleryTile title="Button Variant Example">
-      <NxDropdownButtonVariantExample />
-      <CodeExample content={nxDropdownButtonVariantExampleCode} />
-    </GalleryTile>
-
     <GalleryTile title="Disabled Example">
       <NxDropdownDisabledExample />
       <CodeExample content={nxDropdownDisabledExampleCode} />
     </GalleryTile>
 
-    <GalleryTile title="Actions & Lists Example">
-      <NxDropdownMixedExample />
-      <CodeExample content={nxDropdownMixedExample} />
-    </GalleryTile>
-
     <GalleryTile title="Links Example">
```

Comment:
Would this ticket be a good time to update the examples on this page to use the new `GalleryExampleTile`?

Reply:
> Would this ticket be a good time to update the examples on this page to use the new `GalleryExampleTile`?

For the sake of velocity I would prefer not to at this point, but we could.

Reply:
> For the sake of velocity I would prefer not to at this point, but we could.

We are intending to port all of them before 1.0, correct?

Reply:
> > For the sake of velocity I would prefer not to at this point, but we could.
> 
> We are intending to port all of them before 1.0, correct?

Ideally, but I think clearing out the existing v1 issues is higher priority